app/controllers/service_controller.rb

```diff
@@ -385,7 +420,13 @@ def replace_right_cell(options = {})
       elsif params[:display]
         r[:partial => 'layouts/x_gtl', :locals => {:controller => "vm", :action_url => @lastaction}]
       elsif record_showing
-        r[:partial => "service/svcs_show", :locals => {:controller => "service"}]
+        if action
+          partial_locals[:item_id] = @item.id
```

Comment:
partial_locals is not defined here (broken on master)

don't know about Gaprindashvili

ping @lgalis